frame/transaction-payment/asset-conversion-tx-payment/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "pallet-asset-conversion-tx-payment"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+description = "Pallet to manage transaction payments in assets by utilising the assets conversion pallet"
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+# Substrate dependencies
+sp-core = { version = "21.0.0", default-features = false, path = "../../../primitives/core" }
+sp-io = { version = "23.0.0", default-features = false, path = "../../../primitives/io" }
+sp-runtime = { version = "24.0.0", default-features = false, path = "../../../primitives/runtime" }
```

Comment:
I doubt if all of these deps are needed -- `cargo-machete` can help you find.

Reply:
Good spot - checked and `sp-core` and `sp-io` can be put as dev deps.

Reply:
Thanks!